mentat/mentat_prompt_session.py

```diff
@@ -42,15 +60,142 @@ def get_suggestion(self, buffer: Buffer, document: Document) -> Suggestion | Non
             return super().get_suggestion(buffer, document)
 
 
+### TODO
+# 4. Decide on what words to auto complete (right now it's everything)
+#   a. Use ctags to add types (e.g. function, class) to completions
+# 5. Use ctags if available, fallback to pygments lexer if not
+# 6. Consider fuzzy matching completions
```

Comment:
I love fuzzy matching in general. I guess the downside could be too many options popping up for everything you type?

Rift has an `@` syntax for referencing files, maybe something like that could be good

Reply:
You can limit the number of completions shown. Prompt Toolkit has a built-in fuzzy matcher that's pretty fast too.

Yea I like the `@` directive for Rift and Cursor. Happy to make that character show file completions specifically.